cmd/ctr/commands/containers/containers.go

```diff
@@ -123,20 +125,72 @@ var listCommand = cli.Command{
 			return nil
 		}
 		w := tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
-		fmt.Fprintln(w, "CONTAINER\tIMAGE\tRUNTIME\t")
+		fmt.Fprintln(w, "CONTAINER\tIMAGE\tKIND\t\tCREATED\tPODINFO\tCMD\t")
+
+		var cPrintArray cInfoPrintSlice
+		local, errLocal := time.LoadLocation("Local")
+		if errLocal != nil {
+			return errLocal
+		}
 		for _, c := range containers {
 			info, err := c.Info(ctx, containerd.WithoutRefreshedMetadata)
 			if err != nil {
 				return err
 			}
+			spec, err := c.Spec(ctx)
+			if err != nil {
+				return err
+			}
+			cmds := "-"
+			if len(spec.Process.Args) > 0 {
+				cmds = strings.Trim(strings.Join(spec.Process.Args, " "), " \t\r\n")
+			}
+			kind := "-"
+			if v, OK := info.Labels["io.cri-containerd.kind"]; OK {
+				kind = v
+			}
+			podName := "-"
+			if v, OK := info.Labels["io.kubernetes.pod.name"]; OK {
+				podName = v
+			}
+			podNS := "-"
+			if v, OK := info.Labels["io.kubernetes.pod.namespace"]; OK {
+				podNS = v
+			}
+
+			podUID := "-"
+			if v, OK := info.Labels["io.kubernetes.pod.uid"]; OK {
+				podUID = v
+			}
+			podInfoStr := podName + "|" + podNS + "|" + podUID
```

Comment:
I think it is better to use a tool like [crictl](https://github.com/kubernetes-sigs/cri-tools/tree/master/cmd/crictl) to get k8s specific information, and keep `ctr` k8s-agnostic.

Reply:
Thanks for your suggestion, we will modify it later.